### ISSUE TITLE
Wrong generation of @createdAt directive with custom field

### ISSUE THREAD

**Issue by impowski** [prisma/prisma#4258]
**Describe the bug**

**To Reproduce**
1. Create a data model like so:
```
type User {
  id: ID! @id
  name: String!
  registeredAt: DateTime! @createdAt
}
```
2. Then run `prisma generate`
3. Create some resolver where you will utilize a `createUser` method
3. Start the project
4. Server will give an error on `registeredAt` field

**Expected behavior**
`registeredAt` field should work as `createdAt` be a read-only and in `prisma-client` generated file should be a `DataTimeOutput` instead of `DataTimeInput`

**Versions (please complete the following information):**
 - Connector: [`Postgres`]
 - Prisma Server: [e.g. `prisma@beta`]
 - `prisma` CLI: [`prisma/1.30.0-beta.3 (darwin-x64) node-v11.12.0`]
 - OS: [`OS X Mojave`] 
 - `prisma-client`, `prisma-binding`, `graphqlgen`

**Comment by divyenduz** [comment]
https://www.diffchecker.com/jDZJoD83

This is the difference between `prisma generate` and `prisma generate --endpoint` (with datamodel v1.1 API). 

This should be fixed in the `prisma-generate-schema` package.

**Comment by ejoebstl** [comment]
Fixed in #4354.
